apps/pod-issuer-server/src/routes/deel.ts

```diff
@@ -0,0 +1,115 @@
+import express, { Request, Response } from "express";
+import { expressjwt, Request as JWTRequest } from "express-jwt";
+import jwt from "jsonwebtoken";
+import { POD, PODEntries } from "@pcd/pod";
+import {
+  getDeelUserByEmail,
+  getPaystubPODByEmail,
+  savePaystubPOD
+} from "../stores/deel";
+
+const deel = express.Router();
+
+deel.post("/login", (req: Request, res: Response) => {
+  const inputs: { email: string; password: string } = req.body;
+  if (!inputs.email || !inputs.password) {
+    res.status(400).send("Missing query parameter");
+    return;
+  }
+
+  // In practice, get user information from database by email,
+  // Here for demo purposes, we'll allow any email with @zoo.com domain
+  if (!inputs.email.endsWith("@zoo.com")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // In practice, check if the encrypted password match
+  // This is just for demo purposes
+  if (!inputs.password.startsWith("zoo")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // Signing JWT, valid for 1 hour
+  const token = jwt.sign(
+    { email: inputs.email },
+    process.env.DEEL_EDDSA_PRIVATE_KEY!,
+    { algorithm: "HS512", expiresIn: "1h" }
+  );
+  res.send(token);
+});
+
+deel.post(
+  "/issue",
+  expressjwt({
+    secret: process.env.DEEL_EDDSA_PRIVATE_KEY!,
+    algorithms: ["HS512"]
+  }),
+  async (req: JWTRequest, res: Response) => {
+    const email = req.auth?.email;
+    if (!email) {
+      res.status(401).send("Unauthorized");
+      return;
+    }
+
+    const inputs: {
+      semaphoreCommitment: string;
+    } = req.body;
+
+    if (!inputs.semaphoreCommitment) {
+      res.status(400).send("Missing query parameter");
+      return;
+    }
+
+    try {
+      // We already issued paystub POD for this user, return the POD
+      const podStr = await getPaystubPODByEmail(email);
+      if (podStr !== null) {
+        const pod = POD.deserialize(podStr);
+        const owner = pod.content.asEntries().owner.value;
+        if (owner !== BigInt(inputs.semaphoreCommitment)) {
+          res
+            .status(400)
+            .send(
+              "Already issued POD for this user, but Semaphore Commitment doesn't match."
+            );
+          return;
+        }
+        res.status(200).json({ pod: podStr });
+        return;
+      }
+
+      const user = getDeelUserByEmail(email);
+      if (user === null) {
+        res.status(404).send("User not found");
+        return;
+      }
+
+      // For more info, see https://github.com/proofcarryingdata/zupass/blob/main/examples/pod-gpc-example/src/podExample.ts
```

Comment:
I suggest a comment here explaining how this fits into the flow of the app.  I.e. this is case where we haven't issued a POD for the user yet, so need to issue a new one, and save it for future use.